### ISSUE TITLE
No Summary Column For 0 Articles

### ISSUE THREAD

**Issue by heobu** [8hk/AEK#91]
When there are no articles for a combination, then the column does not seem to be shown at the moment.

This can be confusing. It would probably be better to have the columns with 0 articles in them.

**Comment by 8hk** [comment]
I think we need to think about the  edge cases like 4+keywords and 4+dimension. System should show more than 100+ column for total combinations which would be so hard to follow for user. That's why I believe we should not open new column for empty result combination. Instead we can add a memo or something similar like that "those combination has no result" instead opening a new column.

**Comment by heobu** [comment]
Agreed. Moreover, maybe we can make all the columns expand/collapse by clicking on the top part indicating which combination the column is about. Or only make a number of them expanded by default and the user can expand the rest by clicking on them.

**Comment by 8hk** [comment]
Backend appending empty list into the response and frontend parsing-showing the list properly.

![1](https://user-images.githubusercontent.com/39605552/104842892-d387ba80-58d8-11eb-86de-ed72ed4b8540.PNG)
![2](https://user-images.githubusercontent.com/39605552/104842893-d5517e00-58d8-11eb-8458-d9e42c680981.PNG)
![3](https://user-images.githubusercontent.com/39605552/104842896-d682ab00-58d8-11eb-903b-b784fc9bfbd3.PNG)
![4](https://user-images.githubusercontent.com/39605552/104842897-d7b3d800-58d8-11eb-987f-40b86122d030.PNG)

**Comment by heobu** [comment]
Great! Approved the changes too. So we can merge and close this issue.
